subgraph/CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added `createTokenCalls`, `createERC20TokenCalls`, `createWrappedERC20TokenCalls`, `createERC1155TokenCalls` and `createERC721TokenCalls` mocks.
+- Added `createGetProposalCall` and `createTotalVotingPowerCall` mocks.
```

Comment:
Does not belong to commons

Reply:
agreed. these belong into the tbc token voting repo